Add file-explorer-markdown-titles plugin

Dyldog opened the pull request:
<!--- Delete this section if submitting a theme -->
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/Dyldog/file-explorer-markdown-titles

## Release Checklist

<!--- Confirm that you have done the following before submitting your plugin -->
- [x] I have tested this on macOS
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] README clearly describes the plugins purpose and provides clear usage instructions.
- [x] I have added a license in the LICENSE file.

lishid commented:
Could you clean out the plugin a bit more from the original forked code? I'm seeing references to things like "count" and lots of tweak to CSS styles that doesn't seem to belong to what your plugin should be doing.

https://github.com/Dyldog/file-explorer-markdown-titles/blob/main/src/path.d.ts
Don't use `'path'` because then your plugin will only work on desktop.

Dyldog commented:
@lishid V lazy of me 😅 Summary of the updates I made:

- [x] Renamed file-count files to file-title
- [x] Removed path.d.ts
- [x] Removed file-count styles
- [x] Removed unused helpers

Dyldog commented:
@lishid Sorry, I didn't realise I needed to add my plugin files to the release. I've done that now. Anything else I need to do?
